tree/ntuple/v7/inc/ROOT/RField.hxx

```diff
@@ -192,6 +192,9 @@ public:
       void Read(NTupleSize_t globalIndex) { fField->Read(globalIndex, fObjPtr.get()); }
       void Read(RClusterIndex clusterIndex) { fField->Read(clusterIndex, fObjPtr.get()); }
       void Bind(std::shared_ptr<void> objPtr) { fObjPtr = objPtr; }
+      /// In contrast to copying the RValue and calling Bind, where the caller needs to provide an object pointer,
+      /// Clone() lets the field create a new object pointer.
+      RValue Clone() { return fField->GenerateValue(); }
```

Comment:
I think `Clone` is not an ideal name because parts of `this`' value are explicitly not cloned. I don't really have a good naming proposal, also not really happy with `CreateNewValue`...

Reply:
How about `std::shared_ptr<void> CloneObjectPtr {...}` or `CloneObject()`?

Reply:
But we're exactly *not* cloning the underlying object, or am I missing something? `GenerateValue` / `CreateValue` will create a new `RValue` with a newly created object pointer...

Reply:
We discussed offline and came up with the `EmplaceNew` alternative that is now implemented.